packages/enzyme/package.json

```diff
@@ -39,6 +39,7 @@
     "object.assign": "^4.0.4",
     "object.entries": "^1.0.4",
     "raf": "^3.3.2",
+    "rst-selector-parser": "^2.2.0",
     "uuid": "^3.1.0"
```

Comment:
it looks like we don't need this dependency anymore. is that accurate?

Reply:
Yes! I'll remove it.